satpy/etc/readers/viirs_gran.yaml

```diff
@@ -0,0 +1,30 @@
+reader:
+  description: Generic JPSS VIIRS Products (Granule) Reader
+  name: viirs_gran
```

Comment:
Not sure how I missed this, but the reader should probably be named `viirs_edr_gran` (the name in the YAML and the name of the YAML, python name change would be nice but optional). Thoughts?

Reply:
Good point, that's definitely a better name. I changed it in the newest commit.